### ISSUE TITLE
KeyError with get_initial on UpdateWithInlinesView inline

### ISSUE THREAD

**Issue by xrmx** [AndrewIngram/django-extra-views#79]
This fails on GET with key 0 because self.initial_extra is empty, removing the get_initial fixes the issue:

```
File "/venv/local/lib/python2.7/site-packages/bootstrap3/renderers.py" in render_forms
  73.         for form in self.formset.forms:
File "/venv/local/lib/python2.7/site-packages/django/utils/functional.py" in __get__
  49.         res = instance.__dict__[self.func.__name__] = self.func(instance)
File "/venv/local/lib/python2.7/site-packages/django/forms/formsets.py" in forms
  133.         forms = [self._construct_form(i) for i in xrange(self.total_form_count())]
File "/venv/local/lib/python2.7/site-packages/django/forms/models.py" in _construct_form
  848.         form = super(BaseInlineFormSet, self)._construct_form(i, **kwargs)
File "/venv/local/lib/python2.7/site-packages/django/forms/models.py" in _construct_form
  576.                 kwargs['initial'] = self.initial_extra[i-self.initial_form_count()]
```

views.py

```
from extra_views import InlineFormSet, UpdateWithInlinesView
from .models import CommentoLavorazione, Lavorazione

class CommentoLavorazioneInline(InlineFormSet):
    model = CommentoLavorazione
    extra = 1

    def get_initial(self, **kwargs):
        return {'creato_da': self.request.user}

class LavorazioneUpdateView(UpdateWithInlinesView):
    model = Lavorazione
    inlines = [CommentoLavorazioneInline]
```

templates:

```
{% for inline_form in inlines %}
{{ inline_form.as_p }}
{% comment %}
{% bootstrap_formset inline_form layout="tabular" %}
{% endcomment %}
{% endfor %}
```

**Comment by jonashaag** [comment]
+1 for adding docs

**Comment by sdolemelipone** [comment]
Addressed in #178.
